modules/ROOT/pages/proc_installing-cli.adoc

```diff
@@ -10,56 +10,56 @@ In order to interact with Tekton from a terminal, you need to install the Tekton
 . Get the `tar.gz`:
 +
 ----
-curl -LO https://github.com/tektoncd/cli/releases/download/v0.4.0/tkn_0.4.0_Darwin_x86_64.tar.gz
+curl -LO https://github.com/tektoncd/cli/releases/download/v0.8.0/tkn_0.8.0_Darwin_x86_64.tar.gz
```

Comment:
Please add the $ and # to the commands for this module.

Reply:
@Preeticp - I have done the changes for the proc_installing_cli. Pls review